com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -93,7 +102,20 @@ def _get_elements(self, url, key, eclass, id=None, name=None):
         return elements
 
     def _get_element_by_id(self, url, key, eclass, id):
-        """Get a single element matching an id"""
+        """Get a single element matching an `id`
```

Comment:
These messages are not appearing in the Pythondoc output I hope, since they are _ methods.